### DISCUSSION TITLE
How to achieve infinite linear scroll with or without using autoplay plugin?

### DISCUSSION THREAD

**Discussion by mrvlab** [davidjerleke/embla-carousel#603] [Help] [answered]
Hi, so i'm currently using the auto play plugin to switch between slides until a user interacts with the prev/next button. As of now i can only change the delay of the transition between transitions as stated in the documentation. But would like to add something like `animation: 10s slide infinite linear` to achieve a marquee like behaviour with infinite loop & nav buttons. 

This is the expected behaviour 
 - Infinite loop.
 - Once hovered or any form of interaction disable Infinite loop.
 - Navigate by pressing prev/next button or drag while in mobile.
 
 Quite like [this](https://codesandbox.io/s/embla-carousel-github-issue-105-i2gji?file=/src/js/EmblaCarousel.js) sandbox (It's **missing** the loop aspect). OBS‼️ can't use this code since it's using an older version of package
 

Has anyone achieved this behaviour or has any clue on how to achieve it within or outside of Embla carousel package.

**Comment by davidjerleke** [comment, marked as answer]
Hi @MarvinxKiyingi,

Thank you for your question. There’s a plugin being developed for this purpose. It’s 80% complete and I just got a donation so hopefully it will be finished soon.

- This will be tracked in issue: #114.

Best,
David

**Reply by mrvlab** [reply]
That's sound awesome! great job!! 🥳 How do I use it, tried jumping to the docs but couldn't find any new plugins there.

**Reply by davidjerleke** [reply]
@MarvinxKiyingi it seems like you missed that it’s under development and therefore not finished. I have some details left to finish before releasing it.

Best,
David

**Reply by mrvlab** [reply]
Oh.. sorry my bad. How would one find out when it's out?

Best, 
Marvin

**Reply by davidjerleke** [reply]
@MarvinxKiyingi you can check the [releases](https://github.com/davidjerleke/embla-carousel/releases) every now and then to see if it's out. And if you see the pull request being merged you would know that it will be released soon:

- #634

Alternatively, you can hit the Watch button --> Custom --> Releases and you will automatically be notified when a new release is out:

<img width="1197" alt="watch" src="https://github.com/davidjerleke/embla-carousel/assets/11529148/9c083bc7-337d-4315-8a90-f8eb80894bdc">


Best,
David

**Reply by mrvlab** [reply]
Cool! thanks 🙏

**Reply by mrvlab** [reply]
Hey @davidjerleke, 

any update on the development of this https://github.com/davidjerleke/embla-carousel/pull/634 ?

Best,
Marvin

**Reply by davidjerleke** [reply]
@MarvinxKiyingi I have single thing left on my todo list [here](https://github.com/davidjerleke/embla-carousel/pull/634/files#diff-2b9e6af1c5a688cf13feb978652223a529325ab1a6f0366d83d52c69eefa2264R21) so it’s pretty close to being merged. But I’ve been busy fixing some bugs and got a lot of contributions to review recently (which is great of course, more and more devs have been helping me out lately!). Stay tuned.

**Reply by mrvlab** [reply]
Cool! eagerly excited to try it out 👏🏿
